ci/prow/make_config.go

```diff
@@ -1311,6 +1329,7 @@ func addTestCoverageJobIfNeeded(jobDetailMap *map[string][]string, repoName stri
 	if goCoverageMap[repoName] {
 		newJobTypes := append((*jobDetailMap)[repoName], "test-coverage")
 		(*jobDetailMap)[repoName] = newJobTypes
+		delete(goCoverageMap, repoName)
```

Comment:
Please comment explaining why you need to remove the repo from the map.

Reply:
Done.

Reply:
And moved the logic to process other repos into a new function.